a.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+      <link rel="stylesheet" href="STYLE.CSS"/>
+    <title>you tube project</title>
+</head>
+<body>
+    <header>
+    
+        <div class="logo-left">
+        <i class="material-icons">menu</i>
+        <img src="https://th.bing.com/th/id/OIP.WXN1s0dF3oBlOCFVqwSVBgHaEH?w=281&h=180&c=7&r=0&o=5&pid=1.7" width="100"/>
+      
+      </div>
+      <div class="search">
+        <form action="">
+          <input type="text" placeholder="search"/>
+          <button>
+              <i class="material-icons">search</i>
+          </button>
+      </form>
+      <i class="material-icons">mic</i>
+      </div>
+      <div class="header_icons">
```

Comment:
kindly wrap all this section with a <nav> tag

Reply:
- [ ] Hey @jemutakiplagat, Great one here with this review, however, you've pointed out what you want the code author to do but you've not pointed out why they should do what you're proposing. I think it will make your review process more understandable if you say the reason behind your proposed change as this is one key ingredient in code review to bring clarity to the code author.

Reply:
- [ ] Kindly make sure to implement the changes asked by the previous reviewer by giving your review again with keeping the points in mind from [here](https://github.com/jemutakiplagat/review-project/pull/1/files#r862676854).